README.md

```diff
@@ -1,2 +1,73 @@
-# flexvolume-spring-boot-starter
-Spring boot starter to read application configuration from Azure Keyvaults mounted as flexvolumes
+# HMCTS properties volume library
+
+[![Build Status](https://travis-ci.com/hmcts/properties-volume-spring-boot-starter.svg?branch=master)](https://travis-ci.com/hmcts/properties-volume-spring-boot-starter)
+[ ![Download](https://api.bintray.com/packages/hmcts/hmcts-maven/properties-volume-spring-boot-starter/images/download.svg) ](https://bintray.com/hmcts/hmcts-maven/properties-volume-spring-boot-starter/_latestVersion)
+
+This is a Spring Boot _starter_ library to read application configuration properties from files using the file name as key 
+and the file content as value. This can be used in applications that have Azure Keyvaults mounted as flexvolumes, but there 
+are probably other valid uses.
+
+
+## Usage
+Include the module into the dependency list.
+```groovy
+ compile 'uk.gov.hmcts.reform:properties-volume-spring-boot-starter:x.x.x'
+```
+
+To define which directories or files contain Azure Keyvault entries update application.yaml to define the related paths.
+For instance:
+```yaml
+spring:
+  application:
+    name: properties-volume-example
+  cloud:
+    propertiesvolume:
+      enabled: true
+      paths: /kvmnt/this,/kvmnt/that,/kvmount/other
+```
+If a path is a directory all the contained files are loaded.
+For each file found, the file name is the Azure Keyvault key and the file body is the related value. 
+For instance given the following path:
+```
+/kvmnt/draft-store/primary-encryption-key
+```
+a property named as follows would be created:
+
+```
+primary-encryption-key
```

Comment:
this differs to the nodejs implementation in that the keyvault name isn't prefixed,

I would have expected `draft-store` to be prefixed here.

Can the implementations be consistent please?

Reply:
Not necessarily users will mount flexvolumes with the keyvault name. What happens when the mount is: 

'/kvmnt/primary-encryption-key' or '/primary-encryption-key'?